### PR TITLE
release: cut the 7.1.1 release

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

**Comment by SvetoslavTsenov** [comment]
test tns-core-modules@latest

**Comment by SvetoslavTsenov** [comment]
test tns-core-modules#latest

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+<a name="7.1.1"></a>
+## [7.1.1](https://github.com/NativeScript/nativescript-angular/compare/7.1.0...7.1.1) (2018-12-20)
+
+
+### Bug Fixes
+
+* **p-r-o:** needless forward navigation after back inside nested named outlet ([d8a0653](https://github.com/NativeScript/nativescript-angular/commit/d8a0653))
```
Comment (sis0k0) [inline CHANGELOG.md]:
```suggestion
 * **page-router-outlet:** prevent needless forward navigation after back inside nested named outlet ([d8a0653](https://github.com/NativeScript/nativescript-angular/commit/d8a0653))
```

**Comment by SvetoslavTsenov** [comment]
test tns-core-modules#latest android#latest ios@latest

**Comment by rynop** [comment]
New to NativeScript here - how long before you publish new releases to NPM? I know you just made a github release for `7.1.1` less than 24 hours ago but it is `7.1.0` on [npm](https://www.npmjs.com/package/nativescript-angular).  

I'm just trying to understand the process/timing on github release to `npm publish`. Thanks

**Pull request merged**